public/js/whiteboard.js

```diff
@@ -39,6 +44,20 @@ function updateDate() {
     const todaysDate = moment();
     document.querySelector('#date').innerHTML = todaysDate.format('D MMM YY');
     document.querySelector('#time').innerHTML = todaysDate.format('HH:mm');
+
+    const times = SunCalc.getTimes(new Date(), latitude, longitude);
+    const now = Date.now();
+    if (
```

Comment:
Please comment what this `if` statement is checking in plain English? I think it's "if the time is between (a half hour after sunrise) and (a half hour after sunset)" but I'm not positive

Reply:
done